remapping_commands.sh

```diff
@@ -118,7 +121,7 @@ echo '------------------------------3) Data extraction--------------------------
 cd "$TMPDIR"
 samtools sort reads_aligned.bam -o reads_aligned.sorted.bam
 samtools index reads_aligned.sorted.bam
-../reverse_strand.py -i reads_aligned.sorted.bam -p old_ref_alleles.txt -o variants_remapped.vcf
+../reverse_strand.py -i reads_aligned.sorted.bam -p old_ref_alleles.txt -o variants_remapped.vcf -f "$flankingseq"
 
 # Add interval for variant position in bed format (required by getfasta)
 # Reprints all the columns, adding an extra column before the pos column as the pos-1, as bedtools getfasta requires a 
```

Comment:
AS/XS filtering is not implemented in the current version of the code in this PR (just adding this discussion in order to not forget it before closing the ticket)